public_interface/settings.py

```diff
@@ -17,13 +17,19 @@
 
 # don't initialize Sentry if we're in a local dev environment
 if bool(os.environ.get('EB_ENVIRONMENT_NAME')):
-    import sentry_sdk
+    eb_env_name = os.environ.get('EB_ENVIRONMENT_NAME')
+    import boto3
+    eb_client = boto3.client('elasticbeanstalk')
+    eb_env_description = eb_client.describe_environments(
+        ApplicationName='eb-calisphere', EnvironmentNames=[eb_env_name])
+    eb_app_version = eb_env_description['Environments'][0]['VersionLabel']
 
+    import sentry_sdk
```

Comment:
I think it's fine to import boto3 and sentry_sdk at the top of settings with all the other imports. The only places I'll do a conditional import like this is if there's actually going to be an error.